projects/storefrontlib/src/cms-components/misc/message/message.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { GlobalMessageType } from '@spartacus/core';
+import { ICON_TYPE } from '@spartacus/storefront';
+
+@Component({
+  selector: 'cx-message',
+  templateUrl: './message.component.html',
+})
+export class MessageComponent {
+  @Input()
+  text: string;
+
+  @Input()
+  type: GlobalMessageType;
+
+  @Output()
+  clear: EventEmitter<void> = new EventEmitter();
```

Comment:
I wonder if that should be clear or close

Reply:
close is better